src/LASlib/lasreader.cpp

```diff
@@ -23,7 +23,7 @@
     implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
 
   CHANGE HISTORY:
-
+    3 February 2017 -- by Florian de Boissieu -- L1798,1813 added parse_str because of strange problem with overloaded functions
```

Comment:
I know that there is a strange problem with overloaded functions. I'm ok with that. But why you needed a new function.